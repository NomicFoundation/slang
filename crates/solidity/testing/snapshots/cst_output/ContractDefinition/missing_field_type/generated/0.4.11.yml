# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ contract Test {                                                                  │ 0..15
  2  │   field;                                                                         │ 16..24
  3  │ }                                                                                │ 25..26

Errors: # 1 total
  - >
    Error: Expected '.' or '/' or '[' or 'c' or 'i' or 'o' or 'p'.
       ╭─[crates/solidity/testing/snapshots/cst_output/ContractDefinition/missing_field_type/input.sol:2:8]
       │
     2 │   field;
       ·        ┬  
       ·        ╰── Found ';'.
    ───╯

Tree: null
