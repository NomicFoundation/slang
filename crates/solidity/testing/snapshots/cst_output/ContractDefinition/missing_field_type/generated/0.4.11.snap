# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Error: Expected '.' or '/' or '[' or 'c' or 'i' or 'o' or 'p'.
   ╭─[crates/solidity/testing/snapshots/cst_output/ContractDefinition/missing_field_type/input.sol:2:8]
   │
 2 │   field;
   ·        ┬  
   ·        ╰── Found ';'.
───╯

Tree: null
